lib/cabal.dependencies.project

```diff
@@ -11,14 +11,6 @@ source-repository-package
   tag: fe4d9e4ec01ba7caf8053d6888ec2e7f89fad874
   --sha256: 19fbwcmdmb9w34cp19r2j4qywhnjmxxdv4rwci29pzbvgbnnjdia
 
-if !arch(javascript)
-  source-repository-package
-    type: git
-    location: https://github.com/mpickering/haskell-filesystem.git
-    tag: 2eb26717e986442796d703a80869e6826a10191e
-    subdir: system-fileio system-filepath
```

Comment:
I'm not sure why this was added but the upstream PR has been merged https://github.com/fpco/haskell-filesystem/pull/27/